### PR TITLE
Show more helpful error if rustfmt is not in PATH.

### LINKED ISSUE #1071: No such file or directory
On linux I get the following error when using cargo fmt after pulling updates from a github repository:

```shell
No such file or directory (os error 2)
usage: cargo fmt [options]
...
```

I don't know how to debug this any further. It would be nice if rustfmt would return a better error when a file is not found (like the name of the file, and where is rustfmt getting that file name from).

### PR DESCRIPTION
**Pull request by johannhof**
This fixes #1071.

### CODE REVIEW

File: src/bin/cargo-fmt.rs [hunk 1]
```diff
@@ -204,6 +204,16 @@ fn format_files(files: &Vec<PathBuf>,
         .stdout(stdout)
         .args(files)
         .args(fmt_args)
-        .spawn());
+        .spawn()
+        .map_err(|e| {
+            match e.kind() {
+                std::io::ErrorKind::NotFound => {
+                    std::io::Error::new(std::io::ErrorKind::Other,
+                                        "Could not run rustmft, please make sure it is in your \
```
Comment (nrc) [inline src/bin/cargo-fmt.rs]: typo: rustmft

**Comment by nrc** [comment]
Thanks for the PR! Looks good, just a typo to fix before merge.

**Comment by johannhof** [comment]
@nrc oops, updated :)

**Comment by nrc** [comment]
Thanks!

**Pull request merged**